modules/table/src/test/java/org/apache/ignite/internal/table/distributed/raft/snapshot/outgoing/OutgoingSnapshotsManagerTest.java

```diff
@@ -59,15 +67,22 @@ void emptyOngoingSnapshotsIfNoSnapshotWasRegistered() {
     }
 
     @Test
-    void registersSnapshot() {
-        OutgoingSnapshot snapshot = mock(OutgoingSnapshot.class);
-        doReturn(partitionKey).when(snapshot).partitionKey();
+    void startsSnapshot() {
+        MvPartitionStorage mvPartitionStorage = mock(MvPartitionStorage.class);
+
+        when(partitionAccess.partitionKey()).thenReturn(partitionKey);
+        when(partitionAccess.mvPartitionStorage()).thenReturn(mvPartitionStorage);
+        when(partitionAccess.txStatePartitionStorage()).thenReturn(mock(TxStateStorage.class));
+
+        when(mvPartitionStorage.committedGroupConfiguration()).thenReturn(mock(GroupConfiguration.class));
+
+        OutgoingSnapshot snapshot = new OutgoingSnapshot(UUID.randomUUID(), partitionAccess);
 
         manager.startOutgoingSnapshot(UUID.randomUUID(), snapshot);
```

Comment:
Do you test that `startOutgoingSnapshot` does not throw any exceptions?

Reply:
Sure, this was the intention. Thanks, added the assertion